### PR TITLE
feat: add chart wizard component spec and tests

### PR DESCRIPTION
**Pull request by inikolova**
Part of https://github.com/telerik/kendo-themes-private/issues/252

Design file: https://www.figma.com/design/17BuimwXMpooI1GIQn7vNF/Chart-Addon-with-Data-(Grid)-Integration-%23989?node-id=128-17600&m=dev

### CODE REVIEW

File: packages/bootstrap/scss/chart-wizard/_variables.scss
Comment (epetrow) [inline packages/bootstrap/scss/chart-wizard/_variables.scss:1]: All variables should be prefixed with `$kendo-`

File: packages/bootstrap/scss/chart-wizard/_variables.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
+// Chart wizard
+
+/// The color of the aria around the chart type icon.
```
Comment (epetrow) [inline packages/bootstrap/scss/chart-wizard/_variables.scss:3]: There is a typo here -- should be area instead of aria ( check all area related variable descriptions )

File: packages/classic/scss/chart-wizard/_variables.scss
Comment (epetrow) [inline packages/classic/scss/chart-wizard/_variables.scss:1]: Same thing here - all variables are missing a prefix and aria should be area

File: packages/default/scss/chart-wizard/_layout.scss
Comment (epetrow) [inline packages/default/scss/chart-wizard/_layout.scss:1]: Missing variable prefix

File: packages/default/scss/chart-wizard/_theme.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
+@mixin kendo-chart-wizard--theme-base() {
+    .k-icon-background-area {
+        color: $chart-wizard-icon-area-color;
+        background-color: $chart-wizard-icon-area-bg;
+
+        .k-selected & {
+            box-shadow: inset 0 0 1px $chart-wizard-icon-area-color;
```
Comment (epetrow) [inline packages/default/scss/chart-wizard/_theme.scss:7]: Maybe all of this could be extracted to a variable?

File: packages/default/scss/chart-wizard/_variables.scss
Comment (epetrow) [inline packages/default/scss/chart-wizard/_variables.scss:1]: Add variable prefix and change aria to area in the variable description

File: packages/fluent/scss/chart-wizard/_layout.scss
Comment (epetrow) [inline packages/fluent/scss/chart-wizard/_layout.scss:1]: Missing variable prefix and also add css variables similar to how the rest of the components in Fluent are implemented

File: packages/fluent/scss/chart-wizard/_theme.scss
Comment (epetrow) [inline packages/fluent/scss/chart-wizard/_theme.scss:1]: Add variable prefix and expose css variables

File: packages/fluent/scss/chart-wizard/_variables.scss
Comment (epetrow) [inline packages/fluent/scss/chart-wizard/_variables.scss:1]: Missing variable prefix

File: packages/html/src/chart-wizard/chart-wizard-splitter.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    props: KendoSplitterProps &
+        React.HTMLAttributes<HTMLDivElement>
+) => {
+    const {
+        children,
+        ...other
+    } = props;
+
+    return (
+        <Splitter style={{ display: "flex", height: "100%", width: "100%", overflow: "hidden" }}
```
Comment (epetrow) [inline packages/html/src/chart-wizard/chart-wizard-splitter.tsx:22]: Could we scope these styles through `k-chart-wizard-splitter` instead of adding them inline?

File: packages/material/scss/chart-wizard/_variables.scss
Comment (epetrow) [inline packages/material/scss/chart-wizard/_variables.scss:1]: Add variable prefix and change aria to area in the variable descriptions

**Review by epetrow — CHANGES_REQUESTED** [review]

**Comment by epetrow** [comment]
Also some tests are breaking -- as far as I could see that is related to passing null to `formButtons`.

**Review by epetrow — APPROVED** [review]
LGTM
